src/app/services/app-config.service.ts

```diff
@@ -19,4 +19,8 @@ export class AppConfigService {
     if ('serviceURL' in environment) return environment.serviceURL;
     return window.location.toString();
   }
+
+  getAllURLs(): { [key: string]: string } {
+    if ('allServers' in environment) return environment.allServers;
+  }
```

Comment:
This should have a return type - but also is functionally equivalent to just `return environment.allServers;` since it has no else branch and therefore would return `undefined` if `environment` doesn't have an `allServers` key. Is this intended?